transport/transport-runtime/src/androidTest/java/com/google/android/datatransport/runtime/scheduling/persistence/TestEventStoreModule.java

```diff
@@ -24,13 +24,15 @@
 @Module
 public abstract class TestEventStoreModule {
   private static final long MAX_DB_STORAGE_SIZE_IN_BYTES = 10 * 1024 * 1024;
+  private static final long MAX_DB_GLOBAL_STORAGE_SIZE_IN_BYTES = 12 * 1024 * 1024;
```

Comment:
I think it would be clearer to have a dedicated limit value of 2MB with a name along the lines of `maxHealthMetricsStorageSize`, wdyt?